gef.py

```diff
@@ -1392,6 +1392,10 @@ def is_branch_taken(self, insn):               pass
     @abc.abstractmethod
     def get_ra(self, insn, frame):                 pass
 
+    @property
```

Comment:
Should I just make this a class attribute? Will test if it speeds things up, since it's 'called' on every register print.